main.tf

```diff
@@ -158,6 +158,7 @@ module "tls" {
 # Creates a public IP address and an Application Gateway for forwarding internet traffic to the GraphDB proxies/instances
 module "application_gateway" {
   source = "./modules/gateway"
+  count  = var.disable_agw ? 0 : 1
```

Comment:
Follow the formatting of the other modules and leave an empty line between source and count